- mean_time: 0
  stdev_time: 0
  pos_var: 0.1
  rot_var: 0.2
  pose:
    position:
      x: 11.44687080383301
      y: -6.823914527893066
      z: 5.053638935089111
    orientation:
      w: 0.6944591999053955
      x: -1.532717419649998e-08
      y: 1.161289109319341e-08
      z: -0.7195322513580322
- mean_time: 35
  stdev_time: 5
- mean_time: 3
  stdev_time: 0.15
  pos_var: 0.1
  rot_var: 0.5
  pose:
    position:
      x: 11.33335399627686
      y: -6.852222442626953
      z: 5.053638935089111
    orientation:
      w: 0.949495792388916
      x: -0.2891382575035095
      y: -0.0355076901614666
      z: 0.1166031062602997
- mean_time: 5
  stdev_time: 0.5
  pos_var: 0.1
  rot_var: 0.2
  pose:
    position:
      x: 10.69528579711914
      y: -5.602437973022461
      z: 4.434074401855469
    orientation:
      w: 0.7750372886657715
      x: -0.6200490593910217
      y: -0.0761457234621048
      z: 0.0951787456870079
- mean_time: 70
  stdev_time: 10
- mean_time: 2
  stdev_time: 0.1
  pos_var: 0.1
  rot_var: 0.5
  pose:
    position:
      x: 10.69528579711914
      y: -5.602437973022461
      z: 4.434074401855469
    orientation:
      w: 0.4070101976394653
      x: -0.5407271981239319
      y: 0.3121411204338074
      z: 0.666727602481842
- mean_time: 5
  stdev_time: 0.5
  pos_var: 0.3
  rot_var: 0.5
  pose:
    position:
      x: 10.23469352722168
      y: -6.251724720001221
      z: 4.875138282775879
    orientation:
      w: 0.4534505903720856
      x: -0.603954553604126
      y: 0.5338935852050781
      z: 0.3802376985549927
- mean_time: 5
  stdev_time: 0.5
  pos_var: 0.1
  rot_var: 0.2
  pose:
    position:
      x: 10.37275218963623
      y: -6.555939197540283
      z: 4.901471138000488
    orientation:
      w: 0.7044519186019897
      x: -0.0298076868057251
      y: 0.0337786003947258
      z: 0.708321750164032
- mean_time: 55
  stdev_time: 5